pvlib/tests/test_conftest.py

```diff
@@ -19,3 +20,25 @@ def test_fail_on_pvlib_version_pass():
 @fail_on_pvlib_version('100000.0')
 def test_fail_on_pvlib_version_fail_in_test():
     raise Exception
+
+
+# set up to test passing arguments to conftest.fail_on_pvlib_version
+@pytest.fixture()
+def some_data():
+    return "some data"
+
+
+def alt_func(*args):
+    return args
+
+
+deprec_func = deprecated('0.8', alternative='alt_func',
+                         name='deprec_func', removal='0.9')(alt_func)
+
+
+@fail_on_pvlib_version('0.9')
+def test_deprecated_09(some_data):
```

Comment:
following the pattern in the functions above, use version numbers that will never (realistically) need to be changed.